City-of-Helsinki/drupal-asuntotuotanto
# SATT-117: Elastic search update

### Desription

Elasticsearch update from 7.9.2 to 8.4.0 and helfi_platform config updates


### How to test it

- git checkout develop
- make clean down
- git checkout SATT-117-Helfi_platform_update
- make fresh
- make ps
  - check that elastic image is up and running
- login as admin
- go search api indexes https://asuntotuotanto.docker.so/fi/admin/config/search/search-api
  - rebuild track information and re-index both indexes
  - check that both indexes are indexing
- go check that Hitas and Haso listing pages are showing projects
- check that elastic data looks fine localhost:9200/_search?size=1000&pretty=true

!! DO not merge !!
